packages/react-components/src/components/ChatMessageComponentAsEditBox.tsx

```diff
@@ -35,7 +35,7 @@ export type EditBoxProps = {
 /**
  * @private
  */
-export const EditBox = (props: EditBoxProps): JSX.Element => {
+export const ChatMessageComponentAsEditBox = (props: EditBoxProps): JSX.Element => {
```

Comment:
Consider creating a directory for all these together instead:

    src/components/chatmessage
        ChatMessageComponent.tsx
        EditBox.tsx
        ChatBubble.tsx
        FormattedMessageContent.tsx

Also avoids the use of `utils/` directory -- which tends to lose any particular meaning quickly.

Reply:
Created a folder for all the components, converted the util funciton into a react fn component (as that's really what it was)